public/pages/DetectorResults/containers/AnomalyHistory.tsx

```diff
@@ -97,26 +97,24 @@ export const AnomalyHistory = (props: AnomalyHistoryProps) => {
       try {
         setIsLoadingAnomalyResults(true);
         const anomalySummaryResult = await dispatch(
-          searchES(
-            getAnomalySummaryQuery(
+        searchResults(getAnomalySummaryQuery(
               dateRange.startDate,
               dateRange.endDate,
               props.detector.id
-            )
-          )
+            ))
         );
         setPureAnomalies(parsePureAnomalies(anomalySummaryResult));
         setBucketizedAnomalySummary(parseAnomalySummary(anomalySummaryResult));
+
         const result = await dispatch(
-          searchES(
-            getBucketizedAnomalyResultsQuery(
+          searchResults(getBucketizedAnomalyResultsQuery(
```

Comment:
minor: I see this was already like this before, but I feel that overall we should try to avoid making direct calls and using the results directly, since it may cause errors to use those results/responses in downstream functions, etc. This is where redux store can help here in storing data and error messages. Not a blocker obviously

Reply:
we have to use direct calls because we are calling same API in multiple places of same component, but current redux store does not support. We have an issue #23 for this.

Reply:
I see, thanks for clarifying. Another solution may be making reducer functions more fine-grained and/or storing more fine-grained results in the redux store. Will add suggestion to the issue.